tools/frr-reload.py

```diff
@@ -1197,14 +1200,18 @@ def compare_context_objects(newconf, running):
             os.makedirs('/var/log/frr/')
 
         logging.basicConfig(filename='/var/log/frr/frr-reload.log',
-                            level=logging.INFO,
                             format='%(asctime)s %(levelname)5s: %(message)s')
 
     # argparse should prevent this from happening but just to be safe...
     else:
         raise Exception('Must specify --reload or --test')
     log = logging.getLogger(__name__)
 
+    if args.debug:
+        log.setLevel(logging.DEBUG)
+    else:
+        log.setLevel(args.log_level.upper())
```

Comment:
does this translates correctly to the same as `logging.<LEVEL>`?

Reply:
It does, since 2.7.0/3.2: https://github.com/python/cpython/commit/cbb24b35a03f382e560ca4b5fd17bcfe6e67c0f7
eg.
```
Python 3.7.3 (default, Dec 20 2019, 18:57:59)                                  
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.         
>>> import logging
>>> log = logging.getLogger()
>>> log.getEffectiveLevel()
30
>>> log.setLevel("DEBUG")
>>> log.getEffectiveLevel()
10
>>> 
```